Helper/PaymentMethods/GooglePayPaymentMethod.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2022 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+namespace Adyen\Payment\Helper\PaymentMethods;
+
+class GooglePayPaymentMethod extends AbstractWalletPaymentMethod
+{
+    const TX_VARIANT = 'googlepay';
```

Comment:
It could be done, but since going forward Adyen is recommending to use googlepay and because we're not sure of how the transaction variant would look like in the case of paywithgoogle